clues_test.go

```diff
@@ -32,8 +32,8 @@ func mustEquals[K comparable, V any](
 ) {
 	e, g := toMSS(expect), toMSS(got)
 
-	if hasCluesTrace && len(g) > 0 {
-		if _, ok := g["clues_trace"]; !ok {
+	if len(g) > 0 {
+		if _, ok := g["clues_trace"]; !ok && hasCluesTrace {
```

Comment:
automatic clues traces have been removed in favor of following span patterns.  So this moved to reflect that change.  Also, you'll see a lot of `true -> false` changes in the tests; those are all to match this change.  False, in this case, means "dont expect to see a clues_trace value in the map".